examples/binding_affinity_prediction/README.md

```diff
@@ -13,15 +13,63 @@ processed for data artifacts, as additional benchmarking targets.
 - **Atomic Convolutional Networks (ACNN)** [5]: Constructs nearest neighbor graphs separately for the ligand, protein and complex 
```

Comment:
Is it possible to support custom datasets?

Reply:
Implemented